examples/p5js/Handpose/Handpose_Part_Selection/sketch.js

```diff
@@ -0,0 +1,43 @@
+let handpose;
+let video;
+let hands = [];
+
+function setup() {
+  createCanvas(640, 480);
+  video = createCapture(VIDEO);
+  video.size(width, height);
+
+  handpose = ml5.handpose(video, modelReady);
+
+  // This sets up an event that fills the global variable "predictions"
+  // with an array every time new hand poses are detected
+  handpose.on("predict", function(results) {
```

Comment:
```suggestion
  handpose.on("hand", function(results) {
```

Adding in preparation for the update here.